packages/hash/api/src/graphql/resolvers/knowledge/comment/has-text.ts

```diff
@@ -0,0 +1,28 @@
+import { CommentModel } from "../../../../model";
+import { ResolverFn } from "../../../apiTypes.gen";
+import { LoggedInGraphQLContext } from "../../../context";
+import {
+  UnresolvedPersistedCommentGQL,
+  UnresolvedPersistedEntityGQL,
+} from "../model-mapping";
+import { WORKSPACE_TYPES } from "../../../../graph/workspace-types";
+
+export const persistedCommentHasText: ResolverFn<
+  Promise<UnresolvedPersistedEntityGQL[]>,
+  UnresolvedPersistedCommentGQL,
+  LoggedInGraphQLContext,
+  {}
+> = async ({ entityId }, _, { dataSources }) => {
+  const { graphApi } = dataSources;
+  const commentModel = await CommentModel.getCommentById(graphApi, {
+    entityId,
+  });
+  const textEntityModel = await commentModel.getHasText(graphApi);
+
+  // @todo Use getTokens() method from TextModel class
```

Comment:
This reads to me like we already have a `TextModel` class, maybe:
```suggestion
  // @todo implement `TextModel` class so that a `TextModel.getTokens()` method can be used here
```